reactive/filebeat.py

```diff
@@ -193,6 +197,36 @@ def check_filebeat_repo():
     remove_state("filebeat.repo.changed")
 
 
+@when("apt.installed.filebeat")
+@when("config.changed")
+def is_config_valid() -> bool:
+    """Sanity check config options.
+
+    :return: True if all checks pass, False otherwise
+    """
+    log("Executing sanity checks on config values.")
+
+    # sanity check value compliance (e.g., clean_inactive must be greater
+    # than ignore_older + scan_frequency). scan_frequency is no longer a
+    # charm config option but it defaults to 10s and it's set to that in
+    # the charm templates so using that below.
+    scan_frequency = 10
+    ignore_older = config().get("ignore_older")
+    clean_inactive = config().get("clean_inactive")
+
+    if clean_inactive and clean_inactive <= ignore_older + scan_frequency:
+        log(
+            "Invalid config, make sure that the value for 'clean_inactive' "
+            "is greater than 'ignore_older + scan_frequency'."
+        )
+        status.blocked("Invalid config. Please check the logs for details.")
+        return False
+
+    log("Charm config sanity checks passed. Restarting Filebeat")
+    service("restart", "filebeat")
```

Comment:
this should also be unnecessary given the restart at line 99

Reply:
I suggest to remove this line because the function name `is_config_valid` should be a read-only check instead of raise side-effect(restart service).